spec/settings_spec.coffee

```diff
@@ -19,6 +19,12 @@ describe 'Settings', ->
     unset_analytics_object()
     delete @settings
 
+  describe '.flavor', ->
+    it 'has property .flavor', ->
+      expect(@settings)
+        .to.have.property('flavor')
+        .that.is.not.empty
```

Comment:
Let's check the value itself, like in the other specs.
`.that.equals('Skroutz')`